src/postgres-util/src/desc.rs

```diff
@@ -135,18 +265,74 @@ impl Arbitrary for PostgresColumnDesc {
             any::<u32>(),
             any::<i32>(),
             any::<bool>(),
-            any::<bool>(),
         )
             .prop_map(
-                |(name, col_num, type_oid, type_mod, nullable, primary_key)| PostgresColumnDesc {
+                |(name, col_num, type_oid, type_mod, nullable)| PostgresColumnDesc {
                     name,
                     col_num: Some(col_num),
                     type_oid,
                     type_mod,
                     nullable,
-                    primary_key,
                 },
             )
             .boxed()
     }
 }
+
+/// Describes a kye in a [`PostgresTableDesc`].
+#[derive(Debug, Clone, Eq, PartialEq, Serialize, Deserialize, PartialOrd, Ord)]
+pub struct PostgresKeyDesc {
+    /// This key is derived from the `pg_constraint` with this OID.
+    pub oid: u32,
```

Comment:
`postgres_types` includes an `Oid` type alias https://docs.rs/postgres-types/latest/postgres_types/type.Oid.html

Reply:
I had cargo culted this from elsewhere in the file, but address with [postgres-util: describe Oids using tokio_postgres::types::Oid](https://github.com/MaterializeInc/materialize/pull/18403/commits/15898afdfe8063d99b5dc277308e1e8d89743e42)